xknx/core/state_updater.py

```diff
@@ -66,24 +65,41 @@ def parse_tracker_options(tracker_options):
                     tracker_type,
                     update_interval,
                 )
+            if update_interval > MAX_UPDATE_INTERVAL:
+                logger.warning(
+                    "StateUpdater interval of %s to long for %s. Using maximum of %s minutes (1 day)",
+                    tracker_options,
+                    remote_value,
+                    MAX_UPDATE_INTERVAL,
+                )
+                update_interval = MAX_UPDATE_INTERVAL
             return (tracker_type, update_interval)
 
-        tracker_type, update_inteval = parse_tracker_options(tracker_options)
+        async def read_state_mutex():
+            """Schedule to read the state from the KNX bus - one at a time."""
+            async with self._one_by_one:
+                # wait until there is nothing else to send to the bus
+                await self.xknx.telegram_queue.outgoing_queue.join()
```

Comment:
It would probably work without this too and we could remove the xknx reference in the class completely. But I think it is ok to be extra safe for now 😉

Reply:
Yea that's fine. I'd still prefer to update more than one state at a time to not have the user wait extra long. See my other comment.